projects/components/src/table/table.component.ts

```diff
@@ -185,6 +188,9 @@ export class TableComponent
   @Input()
   public columnConfigs?: TableColumnConfig[];
 
+  @Input()
+  public metadata?: FilterAttribute[];
```

Comment:
Strikes me as strange that the table needs this. I guess let me see what data cell renderer is doing with it.

Reply:
https://github.com/hypertrace/hypertrace-core-ui/pull/75/files#r469530453